cloudbuild.yaml

```diff
@@ -8,23 +8,7 @@ steps:
     entrypoint: make
     args:
       - image-push
-    env:
-      - IMAGE_REGISTRY=gcr.io/$PROJECT_ID
-      - GIT_TAG=$_GIT_TAG
-      - EXTRA_TAG=$_PULL_BASE_REF
-      - DOCKER_BUILDX_CMD=/buildx-entrypoint
-  - name: 'gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20221214-1b4dd4d69a'
-    entrypoint: make
-    args:
       - debug-image-push
-    env:
-      - IMAGE_REGISTRY=gcr.io/$PROJECT_ID
-      - GIT_TAG=$_GIT_TAG
-      - EXTRA_TAG=$_PULL_BASE_REF
-      - DOCKER_BUILDX_CMD=/buildx-entrypoint
-  - name: 'gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20221214-1b4dd4d69a'
-    entrypoint: make
-    args:
       - importer-image-push
     env:
```

Comment:
we are missing the IMAGE_REGISTRY variable here.

Reply:
Ah right.

Well, the thing is that importer and debug images will never be in `registry.k8s.io`, whereas the main `kueue` image will.

Having the 2 registries as different variables theoretically allows you to run steps for images located in different places.

Reply:
It's hard to follow, let's keep a single var for now.